src/queue.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+export default class Queue {
+  constructor() {
+    this.storage = []
+  }
+
+  enqueue(element) {
+    return this.storage.push(element)
+  }
```

Comment:
I would not use ```.push``` and the only reason is because, if you want to get the most value out of this project, you want to figure out how to add things to a queue with the least amount of JavaScript methods as possible.